prime-router/src/main/kotlin/tokens/Scope.kt

```diff
@@ -16,26 +15,71 @@ class Scope {
 
         /**
          * Scope that represents prime admin access.
+         * Pattern is <organization>.<receiver-or-sender>.<DetailedScope>
+         * The first value must always be an organization name, or wildcard
+         * The idea is to leave the last field loose; it could be a role or a permission
+         * Examples:
+         * md-phd.*.user -- user level access
+         * md-phd.default.report -- able to submit reports only
+         * md-phd.*.admin
+         * *.*.primeadmin
          */
-        val primeAdminScope = "primeadmin.all.all"
+        val primeAdminScope = "*.*.primeadmin"
 
-        enum class DetailedScope {
-            report, // ability to submit a report
-            all, // all abilities
```

Comment:
A Scope is just a string, of the form `x.y.<DetailScope>`

This is purposely very broad, to allow for future use cases no-one has dreamed of yet.